src/gt4py/gtc/common.py

```diff
@@ -325,7 +325,7 @@ class UnaryOp(GenericNode, Generic[ExprT]):
     op: UnaryOperator
     expr: ExprT
 
-    @root_validator(pre=True)
+    @root_validator(skip_on_failure=True)
```

Comment:
These changes are for the following reason:
- `pre` is not good as the validator runs before fields are checked for validity, therefore the pre-root_validator might emit weird errors if fields are already broken.
- `skip_on_failure` ensures that this test does not run if a field validation fails, because then the `values` dict will not have the failing field at all and again this would result in weird errors.